package.json

```diff
@@ -87,10 +87,11 @@
         "bowser": "1.0.0",
         "corejs-typeahead": "^1.3.1",
         "d3": "3.5.6",
-        "datatables.net": "^1.12.1",
-        "datatables.net-bs": "^1.12.1",
+        "datatables.net": "1.13.4",
+        "datatables.net-bs": "1.13.4",
         "datatables.net-buttons-bs": "2.4.1",
-        "dompurify": "2.3.8",
+        "datatables.net-buttons": "2.3.6",
```

Comment:
New dependency here due to the following shenanigans:
`datatabes.net-buttons-bs` depends on some version of `datatables.net-buttons` >= 2.3.6. Last month, an update was released (version 3.0.0, which fits that criteria) that's incompatible with datatables.net under version 2. An update to the package-lock.json included that change, which breaks compatibility.

I'm not tackling the update from datatables 1 -> datatables 2 in this PR, so this locks the version of datatables.net-buttons for now.